src/components/Slot.tsx

```diff
@@ -57,6 +59,14 @@ function Slot({
     setDummyListState(shuffle(dummyList));
   }, [value, dummyList]);
 
+  const renderDummyList = () => {
+    return dummyListState.map((dummyNumber, slotIndex) => (
+      <div key={slotIndex} className={styles.num} aria-hidden="true">
+        {dummyNumber}
+      </div>
+    ))
+  }
+  console.log(hasInfiniteList)
```

Comment:
Please just remove the console.log, and I'll immediately merge it. Thank you!

Reply:
Removed!